roles.md

```diff
@@ -34,6 +36,72 @@ meetings (summarized above).
+	- work with project leads to [draft and send invoices](https://handbook.hypha.coop/guides.html#creating-an-invoice-for-a-client)
+	- [settle invoice payments](https://handbook.hypha.coop/guides.html#settling-an-invoice-payment)
+- make timely bill payments and ensure organization is in good financial standing
+- keep our books synchronized with bank accounts and match transactions to appropriate categories
+- review quarterly financial reports
+- update financial projections and communicate financial health to members
+- serve as contact person for our accountant and bookkeeper
+- oversee all tax obligations
+	- familiarize with [GST/HST Information for Non-Profit Organizations](https://www.canada.ca/en/revenue-agency/services/forms-publications/publications/rc4081/gst-hst-information-non-profit-organizations.html)
+	- report and remit HST and any other non-payroll taxes as applicable
+	- file annual tax return with our accountant at fiscal year-end
+
+**Authority:**
+- move balances across our internal accounts for amounts <= CAD$10,000
+- work with our accountant and bookkeeper as needed
+
+**Limits:** (on Authority)
+- permission of parent WG needed to:
+    - move balances across our internal accounts for amounts > CAD$10,000
+    - move balances outside of our internal accounts for any reason and amount
+    - make non-routine adjustments to our financial accounts, records, and projections
+    - submit our annual tax return
+- permission of project lead needed to:
+	- alter or send an invoice to a client
+
+**Time Budget:** Up to 2h/week.
+
+**Term:** 6 mo, with renewal. Max 2 terms.
+
+### Finance (Internal)
+
+**Parent WG:** Finance
+
+**Aim:** Fulfill financial responsibilities to employees and contractors, and adhere to associated legal and tax obligations.
+
+**Tasks:**
+- run payroll for each pay period
+	- [verify timesheets](https://handbook.hypha.coop/guides.html#timesheets)
+	- [approve reimbursement of expenses](https://handbook.hypha.coop/guides.html#expense-reimbursement)
+	- [prepare payroll](https://handbook.hypha.coop/guides.html#running-payroll)
+	- ensure payroll taxes are remitted, manually file a nil remittance if a pay period is skipped
+	- archive [Employee Payroll sheet](http://link.hypha.coop/payroll) and [processed payroll records](https://drive.google.com/drive/u/0/folders/1wWo9KqNwWdUK5d-jkApV3id_Y_dpftT9)
+- [onboard employees and contractors](https://handbook.hypha.coop/guides.html#onboarding-checklist), keeping appropriate records
+- adhere to legal requirements around [keeping records](https://www.canada.ca/en/revenue-agency/services/tax/businesses/topics/keeping-records.html) and [handling of Social Insurance Number](https://www.canada.ca/en/employment-social-development/services/sin/reports/code-of-practice/section-3.html), and [our sensitive data practices](https://handbook.hypha.coop/guides.html#sensitive-data)
+
+**Authority:**
+- access to sensitive data on employee records
```

Comment:
We define authority for `external` to move money but done define it for `internal`

Perhaps specify authority is inherited from external? Is there anything in external we don't want to put on internal?

Additionally to the external role authorities
If TransferWise for example has a balance of > 10,000 and we need to move it into the main operating account. 
```suggestion
- all authorities granted to Finance (Internal)
- move balances into the main operating account greater the $10,000
- access to sensitive data on employee records
```
```

Reply:
I specifically want only the external role to move money, so there will not be a case where two ppl are moving money without talking. If internal person needs to move money, aside from payroll, they need to ask the WG.